app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -63,7 +63,7 @@ If you have the right experience, School Direct (salaried) give you QTS with man
 ### Teach First
 Teach First is a 2-year programme leading to a PGDE and QTS. At the end of your first year, you will have QTS. The PGDE is awarded at the end of the second year.
 
-To find and apply for Teach First, you should [visit their website](https://www.teachfirst.org.uk/).
+To find and apply for Teach First, you should [visit the Teach First website](https://www.teachfirst.org.uk/).
```

Comment:
```suggestion
To apply for Teach First, you should [visit the Teach First website](https://www.teachfirst.org.uk/).
```

Reply:
Or "find out more about"?

I don't think that 'find and apply' is relevant for Teach First (unless you know otherwise :-) )

Reply:
would rather 'apply' rather than 'find out more about', just to show that they can't find/apply the usual way (through Find)